keyboards/keebio/iris/keymaps/seth/config.h

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2017 Danny Nguyen <danny@keeb.io>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* Use I2C or Serial, not both */
+
+#define USE_SERIAL
+//#define USE_I2C
+
+/* Select hand configuration */
+
+//#define MASTER_LEFT
+//#define MASTER_RIGHT
+#define EE_HANDS
+
+#undef RGBLED_NUM
+#define RGBLED_NUM 8
+#define RGBLIGHT_ANIMATIONS
+#define RGBLIGHT_SLEEP
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define RGBLED_SPLIT { RGBLED_NUM, 0} // defined to sync animations
```

Comment:
Only have LEDs on one side?

Reply:
I have LEDS on both sides wired over extra data. This is remnant from a discord convo when I mentioned to you how my animations didn't sync. I tried to delete this line but animations still won't sync correctly hence it's inclusion as of latest master.

Reply:
Ah, okay. 

I don't really recall the conversation...

But this didn't work correctly when attaching directly to each controller?

Reply:
I haven't tried it as I didn't wanna mess with the wiring once I got it working with it over TRSS :sweat_smile:

Reply:
lol, okay.